<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysMainWindowClass</class>
 <widget class="QMainWindow" name="SysMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>枪支信息智能管理系统_V0.0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../SysMainWindow.qrc">
    <normaloff>:/SysMainWindow/Resources/login-icon.png</normaloff>:/SysMainWindow/Resources/login-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; color:#548234;&quot;&gt;枪 支 信 息 智 能 管 理 系 统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>218</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_logo">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>141</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>141</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/SysMainWindow/Resources/81logo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>318</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
      <property name="horizontalSpacing">
       <number>50</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_AcManager">
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>账号管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_CodeReview">
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>编码检视</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_InfoManager">
        <property name="minimumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>251</width>
          <height>251</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>信息管理</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../SysMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_AcManager</sender>
   <signal>clicked()</signal>
   <receiver>SysMainWindowClass</receiver>
   <slot>slotAccountManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CodeReview</sender>
   <signal>clicked()</signal>
   <receiver>SysMainWindowClass</receiver>
   <slot>slotCodeReview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_InfoManager</sender>
   <signal>clicked()</signal>
   <receiver>SysMainWindowClass</receiver>
   <slot>slotInfoManager()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAccountManager()</slot>
  <slot>slotCodeReview()</slot>
  <slot>slotInfoManager()</slot>
 </slots>
</ui>
